pyproject.toml

```
@@ -140,6 +139,7 @@ dev = [
   "pytest-asyncio==0.24.*",
   "pytest-cov~=6.1",
   "pytest~=8.3",
+  "pytest-pretty>=1.2.0",
```

Let's use the `~=` operator here, this helps us as a team track our dependencies.
```suggestion
  "pytest-pretty~=1.2",
```